### PR TITLE
blog: headless

### PR DESCRIPTION
**Pull request by irisb1701**
## Description

1. Addition of new .md file that describes steps for headless Raspberry Pi set up via VNC (For those without an extra monitor and keyboard)

2. Placed inside new vi folder 

## Screenshot 

Screen Shot 2019-08-08 at 2 20 07 AM

## Link

https://raw.githack.com/treehouses/treehouses.github.io/headless/#!pages/docs/vi/vi-headless.md

### CODE REVIEW

**Comment by dogi** [comment]
@irisb1701 we should move this into a blog

**Comment by dogi** [comment]
image

please add my pic to the blog @irisb1701 ;)

**Comment by irisb1701** [comment]
@dogi ok! working on it

File: pages/docs/vi/vi-headless.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# First Steps
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: Change title here

File: pages/docs/vi/vi-headless.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Headless Set up
+&nbsp;
+Don’t have an extra keyboard or HDMI cable? Here’s how to do a headless [Treehouses](http://download.treehouses.io/) install on your Pi.
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: This is a blog post separate from the first steps, so it should explain some more about Raspberry Pi and what is headless

Comment (Liew211) [reply pages/docs/vi/vi-headless.md]: Also lowercase treehouses

File: pages/docs/vi/vi-headless.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+&nbsp;
+Here is the list of hardware requirements once again for you to refer to.
+&nbsp;
+#### Hardware requirement review:
+1.  A Raspberry Pi 4
+1.  A power cord for the Pi
+	- Standard micro USB power cord: 2.4A for Pi 3 and Pi 3b+, 1.2A for Pi Zero W, or 15.3W for Pi 4
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: Pi 4 is 3A, Pi 3 B/B+ is 2.5 according to https://www.raspberrypi.org/documentation/faqs/#pi-power

File: pages/docs/vi/vi-headless.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+1.  3 Class 10 microSD cards
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: Class 10 or U1

File: pages/docs/vi/vi-headless.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	- minimal 16GB, but we recommend 32GB or greater
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: mimimum

File: pages/docs/vi/vi-headless.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+1.  A microSD card reader
+	- Check your computer if you have a slot for a card reader otherwise use a USB reader
+1.  Ethernet cable or wifi dongle
+1.  USB keyboard, monitor, and HDMI cable
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: This is a headless tutorial, so these items should not be included

File: pages/docs/vi/vi-headless.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+Give it a few minutes to get started and check for the flashing green light to make sure the image burned correctly. 
+
+## Step 3: Accessing Treehouses 
+
+Now you will open up your terminal or command prompt. To configure your Pi, you will need the IP address. To do this, enter the command:
+~~~
+ping treehouses.local
+~~~
+Once the IP address appears press Ctrl + C.
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: should include example output and other methods of finding pi; if lazy you can just link to [Raspberry Pi documentation](https://www.raspberrypi.org/documentation/remote-access/ip-address.md) or our own first steps finding pi page

File: pages/docs/vi/vi-headless.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+Download [RealVNC](http://realvnc.com/en/connect/download/vnc/) here
+
+Once it finishes downloading, open it up and enter the IP address from earlier in the search bar. Your Pi desktop should appear. 
+
+It may give you an error at first, so ensure VNC is turned on in Treehouses by going back to terminal and entering:
+
+~~~
+sudo -s
+Treehouses vnc on
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: lowercase treehouses everywhere

File: pages/docs/vi/vi-headless.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+reboot
+~~~
+Reboot will restart the Pi so you will need wait a few minutes and then SSH back into it using:
```
Comment (Liew211) [inline pages/docs/vi/vi-headless.md]: screenshots/images would be really helpful in this blog post.

File: pages/docs/vi/vi-headless.md (hunk 7 above)
Comment (Liew211) [reply pages/docs/vi/vi-headless.md]: The details can just be copied from my [Pihole blog post](https://treehouses.io/#!./pages/blog/20190717-pihole.md)

**Review by dogi — APPROVED** [review]

**Review by Liew211 — APPROVED** [review]

**Pull request merged**